### PR TITLE
Upgrade to HL Fabric 1.4.7

### PR DESCRIPTION
**Pull request by fasmat**
This PR upgrades to HL Fabric 1.4.7.

### CODE REVIEW

**Comment by raftAtGit** [comment]
@fasmat thanks a lot for the effort :)

I'd like to accept this and possibly other pull requests, but unfortunately I dont have time to actually verify if any existing functionality is broken. 

That's why I think, automated tests are very important for pull requests. So I created [this issue](https://github.com/APGGroeiFabriek/PIVT/issues/68) with `Help wanted` label. 

It would be really helpful if yourself or any other community member has some time and implement some basic tests for this repo.

File: fabric-kube/docker/fabric-tools/Dockerfile [hunk 1]
```diff
@@ -1,7 +0,0 @@
-FROM raft/hl-fabric-tools:1.4.1
```
Comment (raftAtGit) [inline fabric-kube/docker/fabric-tools/Dockerfile:1]: Why did you delete this file?

Comment (fasmat) [reply fabric-kube/docker/fabric-tools/Dockerfile:1]: I thought it was not used anywhere and is based on an old fabric version 1.4.1. I couldn't update it to 1.4.7, 1.4.3 seems to be the most recent available version.

I assume this is the Dockerfile for the `raft/hl-fabric-tools-extended` image?

Comment (raftAtGit) [reply fabric-kube/docker/fabric-tools/Dockerfile:1]: true, it's for `raft/hl-fabric-tools-extended` image

**Comment by raftAtGit** [comment]
default version is already 1.4.9
